controllers/kogitoserverlessworkflow_controller_test.go

```diff
@@ -63,11 +63,15 @@ func TestKogitoServerlessWorkflowController(t *testing.T) {
 			t.Fatalf("Failed to fetch supposed to exist workflow %v", err)
 		}
 		// Perform some checks on the created CR
-		assert.True(t, afterReconcileWorkflow.Spec.Start == "ChooseOnLanguage")
+
+		assert.True(t, afterReconcileWorkflow.Spec.BaseWorkflow.Start.StateName == "ChooseOnLanguage")
```

Comment:
Maybe we could have a few aliases in the SDK. Nothing to do at the moment, but good food for thought:

`Workflow.Start == "ChooseOnLanguage"`

Reply:
Just assessors of an underlying object. So it should write and read, but not marshaled.